styles/main.scss

```diff
@@ -1,9 +1,44 @@
-$color: blue;
+#rajoyirize-container {
+    font-family: Arial;
+    font-size: 16px;
+    width: 100%;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    overflow: hidden;
+    
+    &.hidden {
```

Comment:
The idea is not to overwrite a "hidden" class the page the plugin is inserted in might already have. But we should add ".hidden" to this declaration.

` &.hidden, .hidden {
        display: none !important;
    }`